services/tenant-ui/frontend/src/components/innkeeper/tenants/deleteTenant/DeleteTenant.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <Button
+      :title="$t('tenants.settings.deleteTenant')"
+      icon="pi pi-trash"
+      class="p-button-rounded p-button-icon-only p-button-text"
+      @click="openModal"
+    />
+    <Dialog
+      v-model:visible="displayModal"
+      :style="{ minWidth: '500px' }"
+      :header="'Delete Tenant'"
+      :modal="true"
+      @update:visible="handleClose"
+    >
+      <ConfirmTenantDeletion
+        :tenant="props.tenant"
+        @success="$emit('success')"
+        @closed="handleClose"
+      />
+    </Dialog>
+  </div>
+</template>
+
+<script setup lang="ts">
+// Types
+import { TenantRecord } from '@/types/acapyApi/acapyInterface';
+
+// Vue
+import { ref } from 'vue';
+// PrimeVue
+import Button from 'primevue/button';
+import Dialog from 'primevue/dialog';
+// Custom Components
+import ConfirmTenantDeletion from './ConfirmTenantDeletion.vue';
+
+defineEmits(['success']);
+
+// Props
+const props = defineProps<{
+  tenant: TenantRecord;
+}>();
+
+const displayModal = ref(false);
+const openModal = async () => {
+  // Kick of the loading asyncs (if needed)
```

Comment:
IMHO these comments aren't useful. I know there is other places with the same comments.